cetlvast/suites/unittest/test_unbounded_variant.cpp

```diff
@@ -19,6 +19,44 @@
 
 // NOLINTBEGIN(*-use-after-move)
 
+namespace cetl
```

Comment:
Maybe these should be moved into a header to allow reuse in other tests, shall the need arise?

Reply:
ok, I'll create helper header.